tasks/main.yml

```diff
@@ -25,7 +25,7 @@
   register: head_plugin_path
 
 - name: Install ES plugins
-  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
+  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}remove {{ item }} && /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
```

Comment:
It does, but as we support installing custom plugins, that are installed with a different name to the directory they are put in to, it means that we would need a check for each plugin and then a mapping from the name you use to install the plugin, and the directory that the plugin goes into.

The task above can be removed!